continuous_integration/environment-3.6.yaml

```diff
@@ -18,7 +19,7 @@ dependencies:
   # - fastparquet
   - h5py
   - pytables
-  - fsspec==0.6.0
+  - fsspec>=0.7.4
```

Comment:
Do the other envs not need to pin a min fsspec like this?

Reply:
All the others had ">=", so would have picked up the latest. I don't know where the failure is coming from, doesn't happen for me even on windows :|